railties/lib/rails/generators/named_base.rb

```diff
@@ -26,6 +26,10 @@ def template(source, *args, &block)
             super
           end
         end
+
+        def js_template(source, destination)
```

Comment:
I think `js_` suggests this shouldn't be the concern of the general named base class. Was this supposed to be in the `Js::Generators::ChannelGenerator` class?